src/agent/driver-test-runner.js

```diff
@@ -127,7 +137,21 @@ export class DriverTestRunner {
           },
         });
       }
-    } else if (!capabilities.atName) {
+    } else if (!atName) {
+      return;
+    }
+    throw new Error(`Unable to ensure proper settings. Unknown atName ${capabilities.atName}`);
+  }
+
+  /**
+   * Used for v1 aria-at tests, "reading" and "interaction" map to various settings based on AT.
+   * @param {"reading" | "interaction"} mode
+   */
+  async ensureMode(mode) {
+    const { atName } = await this.collectedCapabilities;
+    if (atName == 'NVDA') {
```

Comment:
```suggestion
    if (atName === 'NVDA') {
```
not sure if you were trying to coerce types here for a reason I don't understand but if not best to be consistent